components/tooltip/demo/tooltip.html

```diff
@@ -5,6 +5,13 @@
 	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 	<meta charset="UTF-8">
 	<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+	<script>
+		const urlParams = new URLSearchParams(window.location.search);
+		window.D2L = { LP: { Web: { UI: { Flags: { Flag: (key) => {
+			if (key === 'GAUD-7355-tooltip-popover') return (urlParams.get('popover') === 'true');
+			return false;
+		} } } } } };
```

Comment:
Unrelated: this makes me think it might be nice to have a `mockFlag` method or something alongside our `getFlag` that could be used for scenarios like this.

Hmm, or maybe `getFlag` could check the URL params as part of its logic?

Reply:
I've been think about that too. I only hesitated because it means adding more LMS-ism to core. I guess one way or another, we're adding _something_. 🤷‍♂️

Reply:
Yeah exactly... at least this centralizes it all into 1 file of grossness.